mirror-service/pom.xml

```diff
@@ -119,6 +120,32 @@
             <artifactId>quarkus-jacoco</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- Resilience4J -->
+        <dependency>
+            <groupId>io.github.resilience4j</groupId>
+            <artifactId>resilience4j-retry</artifactId>
+            <version>${lib.resilience4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.resilience4j</groupId>
+            <artifactId>resilience4j-ratelimiter</artifactId>
+            <version>${lib.resilience4j.version}</version>
+        </dependency>
```

Comment:
Do we need the rate limiter?

Reply:
I don't think you need the rate limiter here in your code. there is a rate limiter already built into the NVD API library being used.

Reply:
I'm also improving some of the retry logic in the NVD API as I've hit other errors then the occasional "garbage" response from the NVD API.